packages/web-app-admin-settings/src/components/Users/SideBar/EditPanel.vue

```diff
@@ -133,16 +151,38 @@ export default defineComponent({
         valid: true
       }
     })
+
     const groupOptions = computed(() => {
       const { memberOf: selectedGroups } = unref(editUser)
       return props.groups
         .filter((g) => !selectedGroups.some((s) => s.id === g.id))
         .sort((a, b) => a.displayName.localeCompare(b.displayName))
     })
 
-    return { editUser, formData, groupOptions, ...useGraphClient() }
+    const isLoginInputDisabled = computed(
+      () => currentUser.uuid === (props.user.id as PropType<User>)
```

Comment:
For some reason, TS does not recognize the correct `PropType` here... I also had this issue, I have no idea why though. Please use the type like this at least:

```suggestion
      () => currentUser.uuid === (props.user as User).id)
```